srcpkgs/qtile/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'qtile'
 pkgname=qtile
 version=0.22.0
-revision=2
+revision=3
 build_style=python3-pep517
 hostmakedepends="python3-setuptools_scm python3-cairocffi python3-xcffib python3-wheel pkg-config"
 makedepends="python3-devel libffi-devel pulseaudio-devel"
-depends="python3-cairocffi python3-xcffib pango gdk-pixbuf"
+depends="python3-cairocffi python3-xcffib pango gdk-pixbuf pywlroots"
```

Comment:
i don't think this should be specified here, as not everyone using qtile is using the wayland version. Would be better to have a [README.voidlinux](https://github.com/void-linux/void-packages/blob/master/Manual.md#void-specific-documentation) that says to install it for wayland support

Reply:
Honestly i think it should be there, the gnome package for example also installs all wayland dependencies

Reply:
We don't force optional dependencies in this way. If you really want this, add a `qtile-wayland` metapackage for convenience.

Reply:
> We don't force optional dependencies in this way. If you really want this, add a `qtile-wayland` metapackage for convenience.

I have written a commit that alert the user of the pywlroots package in the INSTALL.msg

Reply:
Don't use `INSTALL.msg` in this manner; use `README.voidlinux` as abby recommended. That's the idiomatic way to document information specifically related to our packaging of the software. `INSTALL.msg` is meant as a last-ditch attempt to convey critical information, mostly about potentially breaking changes, even though these messages show up in an already noisy environment.

Reply:
or a subpkg for `qtile-wayland` which is a meta package that pull them in